etc/eslint/rules/typescript.js

```diff
@@ -0,0 +1,2475 @@
+*
+* @name prefer-object-spread
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [prefer-object-spread]{@link https://eslint.org/docs/rules/prefer-object-spread}
+*/
+rules[ 'prefer-object-spread' ] = 'off';
+
+/**
+* Disables the preference for template literals over string concatenation.
+*
+* @name prefer-template
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [prefer-template]{@link https://eslint.org/docs/rules/prefer-template}
+*/
+rules[ 'prefer-template' ] = 'off';
+
+/**
+* Enforces quote properties in object literals to be always present.
+*
+* @name quote-props
+* @memberof rules
+* @type {Array}
+* @default [ 'error', 'always' ]
+* @see [quote-props]{@link https://eslint.org/docs/rules/quote-props}
+*/
+rules[ 'quote-props' ] = [ 'error', 'always' ];
+
+/**
+* Disables the enforcement of the consistent use of either backticks, double, or single quotes.
+*
+* @name quotes
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [quotes]{@link https://eslint.org/docs/rules/quotes}
+*/
+rules[ 'quotes' ] = 'off';
+
+/**
+* Enforces passing the radix parameter to the `parseInt()` function.
+*
+* @name radix
+* @memberof rules
+* @type {string}
+* @default 'error'
+* @see [radix]{@link https://eslint.org/docs/rules/radix}
+*
+* @example
+* // Bad...
+* const bad = parseInt('123');
+*
+* // Good...
+* const good = parseInt('123', 10);
+*/
+rules[ 'radix' ] = 'error';
+
+/**
+* Disables the requirement for `await` in `async` functions.
+*
+* @name require-await
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [require-await]{@link https://eslint.org/docs/rules/require-await}
+*/
+rules[ 'require-await' ] = 'off';
+
+/**
+* Disables the rule that requires or disallows semicolons instead of ASI.
+*
+* @name semi
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [semi]{@link https://eslint.org/docs/rules/semi}
+*/
+rules[ 'semi' ] = 'off';
+
+/**
+* Enforces consistent spacing before function parentheses.
+*
+* @name @typescript-eslint/space-before-function-paren
+* @memberof rules
+* @type {Array}
+* @default [ 'error', { 'anonymous': 'always', 'named': 'never', 'asyncArrow': 'always' }]
+* @see [space-before-function-paren]{@link https://typescript-eslint.io/rules/space-before-function-paren}
+*/
+rules[ '@typescript-eslint/space-before-function-paren' ] = [
+	'error',
+	{
+		'anonymous': 'always',
+		'named': 'never',
+		'asyncArrow': 'always'
+	}
+];
+
+/**
+* Enforces consistent spacing inside parentheses.
+*
+* @name space-in-parens
+* @memberof rules
+* @type {Array}
+* @default [ 'error', 'always' ]
+* @see [space-in-parens]{@link https://eslint.org/docs/rules/space-in-parens}
+*/
+rules[ 'space-in-parens' ] = [ 'error', 'always' ];
+
+/**
+* Enforces consistent spacing after the `//` or `/*` in a comment.
+*
+* @name spaced-comment
+* @memberof rules
+* @type {Array}
+* @default [ 'error', 'always', { 'markers': [ '/' ] }]
```

Comment:
```suggestion
* @default [ 'error', 'always', { 'markers': [ '/' ] } ]
```

and similar changes here and elsewhere.

Reply:
Changed.